php/Exchange.php

```diff
@@ -3439,8 +3443,11 @@ public function init_rest_rate_limiter() {
             'delay' => 0.001,
             'capacity' => 1,
             'cost' => 1,
-            'maxCapacity' => 1000,
+            'maxLimiterRequests' => 1000,
             'refillRate' => $refillRate,
+            'algorithm' => $this->rateLimiterAlgorithm,
+            'windowSize' => $this->rollingWindowSize,
+            'maxWeight' => $this->rollingWindowSize / $this->rateLimit,   // ms_of_window / ms_of_rate_limit
```

Comment:
@caoilainnl in other langs above, I don't see `maxWeight` defined in config. is it a must here in PHP?